src/main/java/step3/view/ResultView.java

```diff
@@ -0,0 +1,28 @@
+package step3.view;
+
+import step3.domain.Cars;
+import step3.service.RacingGame;
+
+public class ResultView {
+    public static void printByAttemps(RacingGame race) {
+        Cars cars = new Cars(race.setCars());
+        for (int i = 0; i < race.getAttemps(); i++) {
+            race.start(cars);
+            startToPrint(i);
+            printByAttemps(cars);
+        }
+    }
+
+    public static void startToPrint(int iter) {
+        if (iter == 0) {
+            System.out.println("실행 결과");
+        }
+    }
+
+    public static void printByAttemps(Cars cars) {
+        for (int i = 0; i < cars.getCars().size(); i++) {
+            System.out.println(cars.getCars().get(i).getRacingLog());
+        }
+        System.out.println();
+    }
```

Comment:
```suggestion
    public static void printByAttemps(RacingLogs racingLogs) {
        for (RacingLog racingLog: racingLogs.getLogList()) {
            System.out.println(format(racingLog));
        }
        System.out.println();
    }
```

ResultView 에서 `domain.Cars` 에 직접 접근하고 있군요.
차라리 Car 에서 `dto/RacingLog` 를 export 해서, 
컨트롤러에서 그렇게 생성한 RacingLog dto 객체를 뷰에게 넘겨주는 방식은 어떠신가요?